src/uint/div.rs

```diff
@@ -17,35 +17,29 @@ impl<const LIMBS: usize> UInt<LIMBS> {
     /// When used with a fixed `rhs`, this function is constant-time with respect
     /// to `self`.
     pub(crate) const fn ct_div_rem(&self, rhs: &Self) -> (Self, Self, u8) {
-        let mut bd = self.bits().saturating_sub(rhs.bits());
+        let mb = rhs.bits();
+        let mut bd = (LIMBS * Limb::BIT_SIZE) - mb;
         let mut rem = *self;
         let mut quo = Self::ZERO;
-
         let mut c = rhs.shl_vartime(bd);
```

Comment:
Doesn't this still shift by a variable number of bits?

Reply:
It depends only on the value of `rhs`, so it should be constant for a given divisor. Previously it depended on the value of `self` as well.

Reply:
Aha!